scripts/nb-tester/test-notebook.py

```diff
@@ -236,19 +241,27 @@ def create_argument_parser() -> argparse.ArgumentParser:
             "quantum resources! Only use this argument occasionally and intentionally." 
         ),
     )
+    parser.add_argument(
+        "--only-submit-jobs",
+        action="store_true",
+        help="Same as --submit-jobs, but also skips notebooks that do not submit jobs to IBM Quantum",
+    )
     return parser
 
 
 if __name__ == "__main__":
     args = create_argument_parser().parse_args()
-
-    paths = map(Path, args.filenames or find_notebooks(submit_jobs=args.submit_jobs))
-    filtered_paths = filter_paths(paths, submit_jobs=args.submit_jobs)
+    paths = map(Path, args.filenames or find_notebooks())
+    filtered_paths = filter_paths(paths, submit_jobs=args.submit_jobs, only_submit_jobs=args.only_submit_jobs)
```

Comment:
Consider having this also take `ExecuteOptions`.

Reply:
Good call. I realised `ExecuteOptions` is now basically the same as the args object so I'm just using that instead.